include/net/net_ip.h

```diff
@@ -40,12 +40,14 @@ extern "C" {
 #define PF_UNSPEC	0	/* Unspecified.  */
 #define PF_INET		2	/* IP protocol family.  */
 #define PF_INET6	10	/* IP version 6.  */
+#define PF_CAN		13	/* CAN protocol family */
```

Comment:
Any hint to add to commit message where the exact number comes from?

Reply:
done